scripts/core/driver.yml

```diff
@@ -116,6 +116,11 @@ etors:
       desc: "version 1.9"
       version: "1.9"
 --- #--------------------------------------------------------------------------
+type: macro
+desc: "Current API version as a macro"
+name: "$X_API_VERSION_CURRENT_M"
+value: "$X_MAKE_VERSION( 1, 9 )"
```

Comment:
One problem I foresee is that this macro will require a manual yaml file update for every new spec version release. We have this same problem with the ze_api_version_t enum and in the past several releases went out missing the manual update and had to be patched later.  

Is their any way we could automate this?

In any case - I don't intend this comment to be a blocker.  Just pointing it out.